docs/data-sources/workspace.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "graalsystels"
```

Comment:
Typo `graalsystels`?

Reply:
I also noticed this typo, I checked it is everywhere in the doc. Let's keep it for consistency for now and plan to correct this typo in another work item.